config/crd/bases/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -178,6 +178,18 @@ spec:
                             default: GITHUB_CLIENT_SECRET
                             type: string
                         type: object
+                      gitlab:
+                        description: Gitlab specific configuration fields that are injected to the backstage instance to allow the plugin to communicate with Gitlab.
+                        properties:
+                          token:
+                            description: Key in the secret with name defined in the 'name' field that contains the value of the authentication token as expected by Gitlab. Required for importing resource to the catalog, launching software templates and more. Defaults to 'GITLAB_TOKEN', empty for not available.
+                            default: gitlab-host
```

Comment:
Shouldn't this be `GITLAB_TOKEN`? Instead of manually updating the CRD in the bundle directory, you can run `make bundle` to populate the bundle CRD with the new fields. This helps keep both CRDs, in sync.